.github/workflows/ci.yaml

```diff
@@ -16,13 +16,13 @@ jobs:
       test-chunks:  ${{ steps['set-test-chunks'].outputs['test-chunks'] }}
       test-chunk-ids: ${{ steps['set-test-chunk-ids'].outputs['test-chunk-ids'] }}
     steps:
-      - uses: webfactory/ssh-agent@v0.5.4
+      - uses: webfactory/ssh-agent@v0.7.0
         with:
           ssh-private-key: ${{ secrets.SSH_GITHUB_SOLIDOC }}
```

Comment:
I'm pretty sure we don't need this anymore, since solidoc repo is now public. I suggest you remove it.
Probably whole ssh-agent can be removed everywhere, since all our dependencies are public now.

Reply:
removed